app/util/analytics.py

```diff
@@ -105,35 +65,9 @@ def get_events(since_timestamp=None, since_id=None):
     :return: The list of events in the given range
     :rtype: list[dict] | None
     """
-    # eventlogs to STDOUT should only be used in debug situations, so don't worry about making events available.
-    if _eventlog_file is None or _eventlog_file.upper() == 'STDOUT':
+    if isinstance(_event_log, EventLog):
+        since_timestamp = float(since_timestamp) if since_timestamp else since_timestamp
+        since_id = int(since_id) if since_id else since_id
+        return _event_log.get_events(since_timestamp=since_timestamp, since_id=since_id)
+    else:
```

Comment:
Should we raise an exception if a user calls `get_events` without initializing first?